user_management/api/tests/test_models.py

```diff
@@ -8,17 +8,17 @@ class TestAuthToken(utils.APIRequestTestCase):
     model = AuthToken
 
     def test_fields(self):
-        fields = self.model._meta.get_all_field_names()
+        fields = get_all_field_names(self.model)
 
-        expected = wipe_id_fields_on_django_lt_17((
+        expected = (
```

Comment:
It is part of incuna_test_utils. @KevinEtienne removed it in incuna/incuna-test-utils@8d652964018e73d85595d1eea686473a7ea07d18